src/server/api/forms/subjectareas.js

```diff
@@ -0,0 +1,32 @@
+const express = require('express');
+
+const router = express.Router();
+const registrar = require('../../services/registrar');
+
+router.post('/', (req, res) => {
+  const { termCode = '19F' } = req.body;
```

Comment:
@tatikamsetti This is still being hardcoded. But the term is being passed here https://github.com/ucla/lti-course-requestor/pull/76/files#diff-6401a680f0e2084051b12f58e88f79e7R84

Use the parameter from the user

Reply:
@rlorenzo The file got messed up. This is not my latest change. I will fix it. Thank you for catching it.